config/externalnamenottested.go

```diff
@@ -335,4 +335,67 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
+	// GuardDuty filters can be imported using the detector ID and filter's name separated by a colon
+	"aws_guardduty_filter": config.TemplatedStringAsIdentifier("name", "{{ .parameters.detector_id }}:{{ .external_name }}"),
+	// aws_guardduty_invite_accepter can be imported using the member GuardDuty detector ID
+	"aws_guardduty_invite_accepter": FormattedIdentifierFromProvider("", "detector_id"),
+	// GuardDuty IPSet can be imported using the primary GuardDuty detector ID and IPSet ID
+	"aws_guardduty_ipset": config.IdentifierFromProvider,
+	// GuardDuty members can be imported using the primary GuardDuty detector ID and member AWS account ID
+	"aws_guardduty_member": config.IdentifierFromProvider,
+	// GuardDuty Organization Admin Account can be imported using the AWS account ID
+	"aws_guardduty_organization_admin_account": FormattedIdentifierFromProvider("", "admin_account_id"),
+	// GuardDuty Organization Configurations can be imported using the GuardDuty Detector ID
+	"aws_guardduty_organization_configuration": FormattedIdentifierFromProvider("", "detector_id"),
+	// GuardDuty PublishingDestination can be imported using the master GuardDuty detector ID and PublishingDestinationID
+	"aws_guardduty_publishing_destination": config.IdentifierFromProvider,
+	// GuardDuty ThreatIntelSet can be imported using the primary GuardDuty detector ID and ThreatIntelSetID
+	"aws_guardduty_threatintelset": config.IdentifierFromProvider,
+
+	// route53recoveryreadiness
+	//
+	// Route53 Recovery Readiness recovery groups can be imported via the recovery group name
+	"aws_route53recoveryreadiness_recovery_group": config.ParameterAsIdentifier("recovery_group_name"),
+	// Route53 Recovery Readiness resource set name can be imported via the resource set name
+	"aws_route53recoveryreadiness_resource_set": config.ParameterAsIdentifier("resource_set_name"),
+
+	// s3control
+	//
+	//
+	"aws_s3_access_point": config.IdentifierFromProvider,
+	// aws_s3_account_public_access_block can be imported by using the AWS account ID
+	"aws_s3_account_public_access_block": config.IdentifierFromProvider,
+	// Access Point policies can be imported using the access_point_arn
+	"aws_s3control_access_point_policy": FormattedIdentifierFromProvider("", "access_point_arn"),
+	// S3 Control Buckets can be imported using Amazon Resource Name (ARN)
+	"aws_s3control_bucket": config.IdentifierFromProvider,
+	// S3 Control Bucket Lifecycle Configurations can be imported using the Amazon Resource Name (ARN)
+	"aws_s3control_bucket_lifecycle_configuration": config.IdentifierFromProvider,
+	// S3 Control Bucket Policies can be imported using the Amazon Resource Name (ARN)
+	"aws_s3control_bucket_policy": config.IdentifierFromProvider,
+
+	// elasticbeanstalk
+	//
+	// Elastic Beanstalk Applications can be imported using the name
+	"aws_elastic_beanstalk_application": config.NameAsIdentifier,
+	// No import
+	"aws_elastic_beanstalk_application_version": config.IdentifierFromProvider,
```

Comment:
Although import docs are missing for the resource, I suspect this resource is using the `name` argument as a naming field. We may also consider configuring accordingly and fix in the tests. What do you think? Even if don't know the exact syntax for the ID, at least we can consider adding two TODOs for: 1. We check the resource ID when we are testing the resource and try to perform the normalization during testing 2. Stating that the API is currently not normalized.